schema/common.json

```diff
@@ -42,6 +42,9 @@
                 "timeout": {
                     "type": "integer",
                     "minimum": 0
+                },
+                "insecure": {
```

Comment:
Should we add parity for environment variables which are considered [obsolete](https://github.com/open-telemetry/opentelemetry-specification/blame/3a13acce1aced655c0957ac1e70e085f6b35bab1/specification/protocol/exporter.md#L70-L73)?

> The environment variables OTEL_EXPORTER_OTLP_SPAN_INSECURE and OTEL_EXPORTER_OTLP_METRIC_INSECURE are obsolete because they do not follow the common naming scheme of the other environment variables. However, if they are already implemented, they SHOULD continue to be supported as they were part of a stable release of the specification.

Reply:
I can remove them from the comment, but the insecure env var is still valid

Reply:
My read of this comment from the spec is that new SDKs shouldn't implement the insecure env vars. If correct, then that interpretation suggests that we shouldn't extend file configuration to include them, no?

Reply:
This isn't indicating these are obsolete: https://github.com/open-telemetry/opentelemetry-specification/blob/3a13acce1aced655c0957ac1e70e085f6b35bab1/specification/protocol/exporter.md?plain=1#L36

Reply:
Oh oh oh I get it. The spec comment is related to plural vs. singular.

Got it. Disregard.